hedera-node/hedera-config/src/test/java/com/hedera/node/config/converter/LongPairConverterTest.java

```diff
@@ -0,0 +1,129 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.config.converter;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import com.hedera.node.config.types.LongPair;
+import com.swirlds.common.config.sources.SimpleConfigSource;
+import com.swirlds.config.api.Configuration;
+import com.swirlds.config.api.ConfigurationBuilder;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class LongPairConverterTest {
+
+    @Test
+    void testNullValue() {
+        // given
+        final LongPairConverter converter = new LongPairConverter();
+        final String value = null;
+
+        // then
+        assertThatThrownBy(() -> converter.convert(value)).isInstanceOf(NullPointerException.class);
+    }
+
+    @Test
+    void testValidValue() {
+        // given
+        final LongPairConverter converter = new LongPairConverter();
+        final String value = "1-2";
+
+        // when
+        final var result = converter.convert(value);
+
+        // then
+        assertThat(result.left()).isEqualTo(1L);
+        assertThat(result.right()).isEqualTo(2L);
+    }
+
+    @Test
+    void testInvalidValueMultipleSemicolon() {
+        // given
+        final LongPairConverter converter = new LongPairConverter();
+        final String value = "1-2-3;";
+
+        // then
+        assertThatThrownBy(() -> converter.convert(value)).isInstanceOf(NumberFormatException.class);
+    }
+
+    @Test
+    void testInvalidValueNoSemicolon() {
+        // given
+        final LongPairConverter converter = new LongPairConverter();
+        final String value = "1";
+
+        // then
+        assertThatThrownBy(() -> converter.convert(value)).isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @Test
+    void testIncludedInConfig() {
+        // given
+        final Configuration configuration = ConfigurationBuilder.create()
+                .withConverter(new LongPairConverter())
+                .withSource(new SimpleConfigSource("testProperty", "1-2"))
+                .build();
+
+        // when
+        final LongPair value = configuration.getValue("testProperty", LongPair.class);
+
+        // then
+        assertThat(value).isNotNull();
+        assertThat(value.left()).isEqualTo(1L);
+        assertThat(value.right()).isEqualTo(2L);
+    }
+
+    @Test
+    void testIncludedInListInConfig() {
+        // given
+        final Configuration configuration = ConfigurationBuilder.create()
+                .withConverter(new LongPairConverter())
+                .withSource(new SimpleConfigSource("testProperty", "1-2"))
+                .build();
+
+        // when
+        final List<LongPair> values = configuration.getValues("testProperty", LongPair.class);
+
+        // then
+        assertThat(values).isNotNull().hasSize(1);
```

Comment:
I think the `isNotNull()` check is not needed because this is automatically done by `hasSize()` (and similar methods).

Reply:
fixed thanks